From 1adad2911c95f66b4a10ca00d0985119c4f4720e Mon Sep 17 00:00:00 2001
From: tracierenea <tracie.perez@mavs.uta.edu>
Date: Tue, 5 Apr 2016 17:56:09 -0500
Subject: [PATCH 02/28] gr-fec: LDPC python function - addressing an
 "undefined" error

Adding verbosity as an argument to this function, to address this error:

File "/home/tracie/pybombs/src/gnuradio/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py", line 478, in inv_mod2
    if verbose:
NameError: global name 'verbose' is not defined
---
 gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py b/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py
index 28eb5d1..c665e9e 100644
--- a/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py
+++ b/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py
@@ -432,7 +432,7 @@ def greedy_upper_triangulation(H, verbose=0):
   if verbose:
     print '--- Error: nonsingular phi matrix not found.'
 
-def inv_mod2(squareMatrix):
+def inv_mod2(squareMatrix, verbose=0):
   """
   Calculates the mod 2 inverse of a matrix.
   """
-- 
2.1.4

